include/perfetto/base/build_configs/bazel/perfetto_build_flags.h

```diff
@@ -21,6 +21,8 @@
 #define GEN_BUILD_CONFIG_PERFETTO_BUILD_FLAGS_H_
 
 // clang-format off
+#define PERFETTO_BUILDFLAG_COUNT sizeof(PERFETTO_BUILDFLAGS) / sizeof(struct PerfettoBuildFlag)
```

Comment:
Can you a) make this a static constant variable instead of #define b) place this under the list of items below.